utils/api_auth.py

```diff
@@ -0,0 +1,37 @@
+import os
+from bcrypt import hashpw, gensalt, checkpw
+
+from pathlib import Path
+import dotenv
+
+env_file = next((Path(__file__).parent.parent / "env").glob("*.env"))
```

Comment:
what does this do?

Reply:
This loads either `.env` or `production.env`  inside of the `env` folder (I adopted this structure from Gamification)